File: logic build problem/easy problems/python/nth_termOfSeries.py
def nth_termOfSeries(n:int)->int: #time - o(n) and space - o(1)
    nthTerm = 0
    for i in range(1,n+1):
        nthTerm += i
    return nthTerm
    
print(nth_termOfSeries(5))


def nth_term(n:int)->int: #time - o(1) and space - o(1)
    return int(n * (n+1) / 2)
    
print(nth_term(5))

File: logic build problem/easy problems/Typescript/nth_termOfSeries.ts
function nth_termOfSeries(n:number):number{ // time - O(n) and space - O(1)
    let nth_term:number = 0 ;
    for(let i:number=1;i<=n;i++){
        nth_term += i ;
    }
    return nth_term ;
}

console.log(nth_termOfSeries(5)) // time -  O(1) and space - O(1)

function nth_term(n:number):number{
    let nth_term:number = (n * (n+1) / 2)
    return nth_term ;
}

console.log(nth_term(5))
